tgui/packages/tgui/styles/interfaces/TechFab.scss

```
@@ -0,0 +1,53 @@
+@use '../base.scss';
+@use '../colors.scss';
+@use '../functions.scss' as *;
+
+$color-default-fill: colors.bg(colors.$primary) !default;
+$background-color: rgba(0, 0, 0, 0) !default;
+$border-radius: base.$border-radius !default;
+$bg-map: colors.$bg-map !default;
+
+//Like candystripe, but in pairs
+.TechFab__Reagent:nth-child(4n+1) {
+  background-color: rgba(0, 0, 0, 0.25);
+}
+
+.TechFab__Reagent:nth-child(4n+2) {
+  background-color: rgba(0, 0, 0, 0.25);
+}
+
+//Rounded corners for all things inside
+.TechFab__ButtonsContainer {
+  border-radius: $border-radius;
+  overflow: hidden;
+}
+
+//Remove rounded corners, so all buttons in a group have corners together
+.TechFab__NumberButton {
+  border-radius: 0;
+  margin-left: 0;
+  margin-right: 0;
+}
+
+//Add a minimal border between grouped buttons
+:not(:last-child) > .TechFab__NumberButton {
+  border-right: solid 1px black;
+}
+
+//Add a thin gradient border between entries
+.TechFab__Recipe {
+  border-image: linear-gradient(to right, 
+    colors.$grey 10%,
+    adjust-color(colors.$grey, $alpha: -1) 40%,
+    adjust-color(colors.$grey, $alpha: -1) 60%,
+    colors.$grey 90%) 30;
+  border-image-width: 1px 0 0 0;
+}
+
+//Accent the design names using a grey shadow
+//Previous alternatives used are left commented out for reference
+.TechFab__RecipeName {
+  //font-size: 1.15em;
+  //font-weight: bold;
+  text-shadow: colors.$grey base.em(1px) base.em(1px);
+}
```

Right, right, newline at EOF
```suggestion
}

```